styles.ts

```diff
@@ -0,0 +1,166 @@
+import { STYLES } from "likeminds_chat_reactnative_integration";
+
+export const setStyles = (gender: string) => {
+  // themeStyling
+  const themeStyles =
+    gender === "male"
+      ? {
+          hue: 10,
```

Comment:
dont need to set hue, if you are setting `primaryColor: "#B7D340",
          secondaryColor: "#B7D340",
          lightBackgroundColor: "hsl(161, 67%, 91%)"`